boilerplate/index.html

```diff
@@ -49,36 +54,51 @@
           <!-- CHANGE ALL THIS -->
           <h1 class="page-header">loklak App Boilerplate</h1>
           <p>This is the loklak app boilerplate. You can copy this file to quick-start your own loklak app!</p>
-		  
-		      <br id="section1"/><br/>
+<<<<<<< HEAD
+
+          <br id="section1"/><br/>
           <h2 class="sub-header">section 1</h2>
-          <p>This boilerplate has three demo sections which are linked with the menu on the left side. You probably don't need sections, then simply delete them.</p>
-          
+          <p>This boilerplate has three demo sections which are linked with the menu on
+             the left side. You probably don't need sections, then simply delete them.</p>
+
           <br id="section2"/><br/>
           <h2 class="sub-header">section 2</h2>
-          <p>To learn how to create an app, please read <a href="https://github.com/fossasia/apps.loklak.org/blob/master/README.md">the apps readme</a></p>
-          
+          <p>To learn how to create an app, please read
+             <a href="https://github.com/fossasia/apps.loklak.org/blob/master/README.md">the apps readme</a></p>
+
           <br id="section2"/><br/>
+=======
+                    <br id="section1"><br>
+          <h2 class="sub-header">section 1</h2>
+          <p>This boilerplate has three demo sections which are linked with the menu on the left side. You probably don"t need sections, then simply delete them.</p>
```

Comment:
Line is longer than allowed. (165 > 129)

*LineLengthBear, severity NORMAL, section `default`.*